src/main/webapp/i18n/de/exam.json

```
@@ -21,8 +21,12 @@
                     "yourPoints": "Deine Punkte",
                     "maxPoints": "Punkte",
                     "bonus": "Bonuspunkte",
-                    "youAchieved": "Du hast {{achievedPoints}} Punkte von möglichen {{normalPoints}} normalen Punkten und {{bonusPoints}} Bonuspunkten erreicht"
-                }
+                    "youAchievedWithBonus": "Du hast {{achievedPoints}} Punkte von möglichen {{normalPoints}} normalen Punkten und {{bonusPoints}} Bonuspunkten erreicht",
+                    "youAchieved": "Du hast {{achievedPoints}} Punkte von möglichen {{normalPoints}} normalen Punkten",
```

I would leave out the 'normal'
```suggestion
                    "youAchieved": "Du hast {{achievedPoints}} von möglichen {{normalPoints}} Punkten",
```